galasa-ui/src/app/actions/getUserAccessTokens.ts

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Do we need a new file for each server action? We could perhaps group related actions together?